src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ParserServiceImplTest {
+    private static final List<String> VALID_DATA = List.of("type,fruit,quantity", "b,banana,20",
+            "b,apple,100", "s,banana,100", "p,banana,13", "r,apple,10",
+            "p,apple,20", "p,banana,5", "s,banana,50");
+    private static final List<String> EMPTY_DATA = List.of();
+    private static final List<String> INVALID_DATA = List.of("putin huylo, russia sucks",
+            "glory to Ukraine", "russia, is, a, terrorist state");

```

Comment:
Are you crazy!!!! You didn't come up something more polite? Change it. It's not seriously.